src/javascript/app/Stores/Modules/Contract/contract-store.js

```diff
@@ -61,63 +61,78 @@ export default class ContractStore extends BaseStore {
     // ----- Actions -----
     // -------------------
     @action.bound
-    drawChart(SmartChartStore, contract_info) {
-        this.forget_id       = contract_info.id;
-        const { date_start } = contract_info;
-        const end_time       = getEndTime(contract_info);
-
-        SmartChartStore.setChartView(contract_info.purchase_time);
+    drawChart() {
+        const {
+            date_expiry,
+            date_start,
+            id,
+            purchase_time,
+            tick_count,
+        } = this.contract_info;
+
+        this.forget_id = id;
+        const end_time = getEndTime(this.contract_info);
+
+        this.smart_chart.setChartView(purchase_time);
         if (!end_time) this.is_ongoing_contract = true;
 
         // finish contracts if end_time exists
         if (end_time) {
-            if (!this.is_ongoing_contract) {
-                SmartChartStore.setStaticChart(true);
+            if (!this.is_ongoing_contract && !(tick_count < 2)) {
+                // set to static chart for contracts with 1 tick_count
```

Comment:
isn't it currently for non one tick contracts?

Reply:
i think it applies for all ongoing contract

Reply:
I mean the comment says `// set to static chart for contracts with 1 tick_count` but the condition is `!is_one_tick_contract`

Reply:
ahh, typo. yes you're right, should be for non one tick contract